### DISCUSSION TITLE
Exposed endpoint /api/auth/_logs raise alert in OWASP scan

### DISCUSSION THREAD

**Discussion by jacklimwenjie** [nextauthjs/next-auth#2562] [Help] [answered]
![Screenshot 2021-08-19 at 7 17 30 PM](https://user-images.githubusercontent.com/48437528/130060268-ac514cbc-d193-4407-bcd8-9b66282114f0.png)

Anyone have any idea what is the purpose of this endpoint and why is it raising alert in OWASP scan?

**Comment by balazsorban44** [comment, marked as answer]
Seems like a false alarm, it doesn't really make sense:

https://github.com/nextauthjs/next-auth/blob/cb844a2436d8637ff06a809a431914ffc9922b6d/src/server/index.js#L262-L277

https://next-auth.js.org/configuration/options#logger

In short, that endpoint receives logs from the client and forwards them to your logging service, if you define one.

From what I can understand, the endpoint would just crash, because the message in the body is not a valid JSON. Try sending the body in a post to that endpoint to test.

**Reply by jacklimwenjie** [reply]
I see, that makes sense. Thank you!
